index2.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles2.css">
+    <title>Concept pagina</title>
+</head>
+
+<body>
+<header class="outer-container">
+
+    <nav class="navbar">
+        <ul>
+            <li><a href="#rolling-stone-title">case studies</a></li>
+            <li><a href="#webdesign-title">services</a></li>
+            <li><a href="#footer-text">contact us</a></li>
+        </ul>
+    </nav>
+
+    <div class="hero-section">
+        <h1>Your website is an investment
+            in the succes of your business.</h1>
+        <p>We are here to help you succeed.</p>
+        <button class="cta.btn" type="button">Lets work together</button>
+    </div>
+
+
+    <img src="assets/squiggle_line_stripe.svg" alt="My Happy SVG" class="swirl">
+    <div class="skewed">skewed</div>
+</header>
+
+
+<main class="main1">
+
+<div class="question-section">
+    <h3>Rekupe is a digital agency in Los Angeles.</h3>
+    &nbsp;&nbsp;
+    <h3>We use strategic UX, SEO optimisation
+        and beautiful visiual designs to help your
+        business grow.</h3>
+    &nbsp;&nbsp;
+    <h3>We see every project as a fresh
+        opportunity, pushing ourselves to create new
+        digital experiences.</h3>
+
+    &nbsp;&nbsp;
+    <p class="question-tekst">Have a question? &zwnj; <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ"> click here</a> &zwnj; to contact us!</p>
+</div>
+
+
+
+    <div class="section-photo1">
+
+        <img src="assets/work-image-01.jpeg" alt="laptop on table" class="laptop-image">
+    <div class="text-section-photo1">
+        <H1 id="rolling-stone-title">Rolling stone</H1>
```

Comment:
Het is heel belangrijk dat je **maximaal 1** H1 heading hebt per webpagina. De rest van de heading H2 t/m H8, ben je vrij om zoveel mogelijk te gebruiken (mits met logische opbouw).